### PR TITLE
Demo updates

### PR DESCRIPTION
**Pull request by jglick**
Nothing exciting.

@reviewbybees

### CODE REVIEW

**Comment by recena** [comment]
:bee: I suppose you tried it :dizzy_face:

**Comment by amuniz** [comment]
:bee:

File: aggregator/demo/Dockerfile [hunk 1]
```diff
@@ -18,21 +17,38 @@ RUN cd /tmp; ln -s /opt/jetty/webapps
...
+ADD repo/src repo-wc/src
+ADD repo/sometests repo-wc/sometests
+RUN chown -R jenkins.jenkins .
 USER jenkins
-RUN /usr/local/bin/plugins.sh /tmp/files/plugins.txt
+RUN echo '<settings><mirrors><mirror><id>central</id><url>http://repo.jenkins-ci.org/simple/repo1-cache/</url><mirrorOf>central</mirrorOf></mirror></mirrors><localRepository>/usr/share/jenkins/ref/.m2/repository</localRepository></settings>' > settings.xml
+RUN /usr/local/maven/bin/mvn -s settings.xml -f repo-wc install && /usr/local/maven/bin/mvn -s settings.xml -f repo-wc/sometests -Dmaven.test.failure.ignore clean install
 
-ADD repo /tmp/files/repo-wc
+ENV JENKINS_UC http://jenkins-updates.cloudbees.com
```
Comment (kzantow) [inline aggregator/demo/Dockerfile]: This URL? (I see you've just moved it, but still)

**Comment by kzantow** [comment]
:bee: AFAICT

**Comment by jglick** [comment]
> I suppose you tried it

Yes!

File: aggregator/demo/Dockerfile (hunk 1 above)
Comment (jglick) [inline aggregator/demo/Dockerfile]: > This URL?

Probably not required, but so far has seemed friendlier for forcing updates. Anyway yeah this is not new in this PR.

**Pull request merged**